data/categories.txt

```diff
@@ -7213,4 +7213,4 @@ sk:Štúdio jogy
 es:Estudio de yoga
 pl:Studio jogi
 nl:Yogastudio
-pt:Estúdio de ioga
\ No newline at end of file
+pt:Estúdio de ioga
```

Comment:
I used Github's web editor. It looks like it added a newline to the end of the file.